druid-shell/src/lib.rs

```diff
@@ -26,7 +26,7 @@
 //! - `DRUID_SHELL_DISABLE_X11_PRESENT`: if this is set and `druid-shell` is using the `x11`
 //! backend, it will avoid using the Present extension.
 
-#![warn(broken_intra_doc_links)]
+#![warn(rustdoc::broken_intra_doc_links)]
```

Comment:
does not compile on 1.51 :/

Reply:
... That's annoying. Fine, I'll just revert it.

Reply:
add a `// TODO(MSRV)` :)